Formula/newsboat.rb

```diff
@@ -14,29 +15,95 @@ class Newsboat < Formula
     sha256 x86_64_linux:  "a8e76829cb365ae43de0e3aa6d442c1a95378abb59a19f6a2f006f0910e3d44b"
   end
 
-  deprecate! date: "2021-11-26", because: "libstfl is deprecated"
-
   depends_on "asciidoctor" => :build
   depends_on "pkg-config" => :build
+  depends_on "python@3.10" => [:build, :test]
   depends_on "rust" => :build
+  depends_on "swig" => :build
   depends_on "xz" => :build
   depends_on "gettext"
   depends_on "json-c"
-  depends_on "libstfl"
 
+  depends_on "ruby"
+
+  uses_from_macos "perl" => [:build, :test]
   uses_from_macos "curl"
   uses_from_macos "libxml2"
   uses_from_macos "libxslt"
+  uses_from_macos "ncurses"
   uses_from_macos "sqlite"
 
+  # Newsboat have their own libstfl fork. Upstream libsftl is gone:
+  # https://github.com/Homebrew/homebrew-core/pull/89981
+  # They do not want to be the new upstream, but use that fork as a temporary
+  # workaround until they migrate to some rust crate
+  # https://github.com/newsboat/newsboat/issues/1881
+  resource("libstfl") do
+    url "https://github.com/newsboat/stfl.git",
+        revision: "c2c10b8a50fef613c0aacdc5d06a0fa610bf79e9"
+    sha256 "c188dcbce5e5804072bd8856e587a14d7e675ae35cb0eb715362032d7e1f9920"
+  end
+
   def install
+    resource("libstfl").stage do
```

Comment:
Can we link this statically? If we can, then we can just install this to `buildpath` so that we don't have to carry it around in the bottle/keg. Static linking will also resolve your `brew linkage` failure.

If we can't link statically, then you can also fix the linkage failure by doing something like
```ruby
ENV.append "LDFLAGS", "-Wl,-rpath,#{libexec}/lib"
```
after the `libstfl` build.